data/operators/highway/services.json

```
@@ -5,6 +5,15 @@
     "exclude": {"generic": ["^services$"]}
   },
   "items": [
+    {
+      "displayName": "Ředitelství silnic a dálnic",
+      "locationSet": {"include": ["cz"]},
+      "tags": {
+        "highway": "services",
+        "operator": "Ředitelství silnic a dálnic",
```

You'll probably want to add `operator:short` here too

```suggestion
        "operator": "Ředitelství silnic a dálnic",
        "operator:short": "ŘSD",
```